meson.build

```diff
@@ -0,0 +1,22 @@
+project(
+  'ctre',
+  'cpp',
+  version: '3.9.0',
+  license: 'Apache-2.0',
+  default_options: ['cpp_std=gnu++20,gnu++17,vc++latest,c++20,c++17'],
```

Comment:
Lists of standards require Meson &ge; 1.3.0.  You can declare `meson_version: '>= 1.3.0'`, or, for broader compatibility, something like:

```meson
default_options: [meson.version().version_compare('>= 1.3.0') ? 'cpp_std=gnu++20,gnu++17,vc++latest,c++20,c++17' : 'cpp_std=gnu++20'],
```